engine/lib/admin.php

```diff
@@ -598,6 +598,8 @@ function _elgg_admin_plugin_screenshot_page_handler($pages) {
 function _elgg_admin_markdown_page_handler($pages) {
 	set_input('plugin_id', elgg_extract(0, $pages));
 	set_input('filename', elgg_extract(1, $pages));
+	
+	elgg_set_context('admin');
```

Comment:
No, push it. Setting admin, replaces the stack. Plugins don't know anymore
if markdown is part of the stack.
On Jun 26, 2015 11:20 AM, "Jeroen Dalsem" <notifications@github.com> wrote:

> In engine/lib/admin.php
> <https://github.com/Elgg/Elgg/pull/8591#discussion_r33340545>:
>
> > @@ -598,6 +598,8 @@ function _elgg_admin_plugin_screenshot_page_handler($pages) {
> >  function _elgg_admin_markdown_page_handler($pages) {
> >  	set_input('plugin_id', elgg_extract(0, $pages));
> >  	set_input('filename', elgg_extract(1, $pages));
> > +	
> > +	elgg_set_context('admin');
>
> do you mean to pop it? there is no markdown context...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8591/files#r33340545>.
>